pages/help/index.tsx

```diff
@@ -0,0 +1,18 @@
+import { withSites } from '~/lib/sites';
+import { pages } from '../help/[page]';
+
+const Help = () => {
+  return <div />;
+};
+
+export default Help;
+export const getServerSideProps = withSites({
+  async getServerSideProps({ sites }) {
+    return {
+      redirect: {
+        permanent: false,
+        destination: `/help/${Object.keys(pages)[0]}`,
+      },
+    };
+  },
+});
```

Comment:
Oh yeah, the reason I have this is because I wanted to redirect /help to the first page (right now /help/add-site-location), and I didn't realize this is related to auth. Let me know if this is something we want to do, otherwise should I just remove index.tsx? Or keep the page empty?